DESCRIPTION

```diff
@@ -18,25 +20,27 @@ Imports:
     car (>= 2.0-24),
     plyr,
     magrittr,
-    dplyr
+    dplyr,
+    pryr,
+    emmeans,
+    stringi,
+    tidyr,
+    stringr
```

Comment:
This is a large number of dependencies to add. Adding dependencies to packages increases the surface area of possible test failures (if the upstream packages break or change functionality, it can cause code/tests here to break). Some of these are necessary (emmeans, I suspect). Some contain convenience functions that are useful when writing quick analysis code but which are probably not necessary in a package and can be replaced with base R equivalents (stringi and stringr for example). I'll look at specific usages and leave more detailed comments at the points where these packages are used.

Reply:
(see my followup comments --- should be able to remove all of these except emmeans)